src/main/rx/xbus.c

```diff
@@ -158,9 +181,68 @@ static void xBusUnpackFrame(void)
 
 }
 
+static void xBusUnpackRJ01Frame(void)
+{
+    // Calculate the CRC of the incoming frame
+    uint16_t crc = 0;
+    uint16_t inCrc = 0;
+    uint8_t i = 0;
+    uint16_t value;
+    uint8_t frameAddr;
+
+    // When using the Align RJ01 receiver with 
+    // a MODE B setting in the radio (XG14 tested)
+    // the MODE_B -frame is packed within some
+    // at the moment unknown bytes before and after:
+    // 0xA1 __ __ 0xA1 12*(High + Low) CRC1 CRC2 + __ __ __
```

Comment:
Hrm, you are probably right. I will start looking into that. If you like to test the CRC stuff, I would be very happy. Could not attach the file here, trying copy&paste for a few packages at least:
Time [s],Value,Parity Error,Framing Error
-4e-06,0xA1,,
3.6e-05,0x1E,,
7.6e-05,0x06,,
0.000116,0xA1,,
0.000156,0x07,,
0.000196,0xD6,,
0.000236,0x08,,
0.000276,0x09,,
0.000316,0x08,,
0.000356,0x00,,
0.000396,0x01,,
0.000436,0xB7,,
0.000476,0x01,,
0.000516,0xB7,,
0.000556,0x0E,,
0.000596,0x48,,
0.000636,0x0E,,
0.000676,0x48,,
0.000716,0x0E,,
0.000756,0x48,,
0.000796,0x0E,,
0.000836,0x48,,
0.00087625,0x0E,,
0.00091625,0x48,,
0.00095625,0x0E,,
0.00099625,0x48,,
0.00103625,0x0E,,
0.00107625,0x48,,
0.00111625,0x4C,,
0.00115625,0xBA,,
0.00119625,0xC8,,
0.00123625,0xE4,,
0.00127625,0xFF,,
0.01399975,0xA1,,
0.01403975,0x1E,,
0.01407975,0x06,,
0.01411975,0xA1,,
0.01415975,0x07,,
0.01419975,0xD6,,
0.01423975,0x08,,
0.01427975,0x09,,
0.01431975,0x08,,
0.01435975,0x00,,
0.01439975,0x01,,
0.01443975,0xB7,,
0.01447975,0x01,,
0.01451975,0xB7,,
0.01455975,0x0E,,
0.01459975,0x48,,
0.01463975,0x0E,,
0.01467975,0x48,,
0.01471975,0x0E,,
0.01475975,0x48,,
0.01479975,0x0E,,
0.01483975,0x48,,
0.01487975,0x0E,,
0.01491975,0x48,,
0.01495975,0x0E,,
0.01499975,0x48,,
0.01504,0x0E,,
0.01508,0x48,,
0.01512,0x4C,,
0.01516,0xBA,,
0.0152,0xC8,,
0.01524,0xE3,,
0.01528,0x7C,,
0.028001,0xA1,,
0.028041,0x1E,,
0.028081,0x06,,
0.028121,0xA1,,
0.028161,0x07,,
0.028201,0xD6,,
0.028241,0x08,,
0.028281,0x09,,
0.028321,0x08,,
0.028361,0x00,,
0.028401,0x01,,
0.028441,0xB7,,
0.028481,0x01,,
0.028521,0xB7,,
0.028561,0x0E,,
0.028601,0x48,,
0.028641,0x0E,,
0.028681,0x48,,
0.028721,0x0E,,
0.028761,0x48,,
0.028801,0x0E,,
0.02884125,0x48,,
0.02888125,0x0E,,
0.02892125,0x48,,
0.02896125,0x0E,,
0.02900125,0x48,,
0.02904125,0x0E,,
0.02908125,0x48,,
0.02912125,0x4C,,
0.02916125,0xBA,,
0.02920125,0xC8,,
0.02924125,0xE1,,
0.02928125,0xC0,,
0.04200475,0xA1,,
0.04204475,0x1E,,
0.04208475,0x06,,
0.04212475,0xA1,,
0.04216475,0x07,,
0.04220475,0xD6,,
0.04224475,0x08,,
0.04228475,0x09,,
0.04232475,0x08,,
0.04236475,0x00,,
0.04240475,0x01,,
0.04244475,0xB7,,
0.04248475,0x01,,
0.04252475,0xB7,,
0.04256475,0x0E,,
0.04260475,0x48,,
0.04264475,0x0E,,
0.04268475,0x48,,
0.04272475,0x0E,,
0.04276475,0x48,,
0.04280475,0x0E,,
0.04284475,0x48,,
0.04288475,0x0E,,
0.04292475,0x48,,
0.04296475,0x0E,,
0.043005,0x48,,
0.043045,0x0E,,
0.043085,0x48,,
0.043125,0x4C,,
0.043165,0xBA,,
0.043205,0xC8,,
0.043245,0xE2,,
0.043285,0x22,,